deployment/docker-compose.selenium.yml

```diff
@@ -33,7 +33,7 @@ services:
     command: |
       bash -c "
         waitforit -host=db -port=5432 -timeout=30
-        pytest -svvv selenium_tests --reruns 3 --reruns-delay 1 --cov-report xml:./coverage.xml --html-report=./report/report.html --randomly-seed=42
+        pytest -svvv selenium_tests --reruns 3 --reruns-delay 30 --cov-report xml:./coverage.xml --html-report=./report/report.html --randomly-seed=42
```

Comment:
why such an increase?

Reply:
Changed by Patryk because of deadlock problem.